pkg/release/release.go

```diff
@@ -19,12 +19,13 @@ package release
 var Base = "4.12.0-0.nightly-2022-10-25-210451"
 
 var Image = map[string]string{
-	"cli":                       "quay.io/microshift/cli:" + Base,
-	"coredns":                   "quay.io/microshift/coredns:" + Base,
-	"haproxy_router":            "quay.io/microshift/haproxy-router:" + Base,
-	"kube_rbac_proxy":           "quay.io/microshift/kube-rbac-proxy:" + Base,
-	"odf_topolvm":               "quay.io/microshift/odf-topolvm-rhel8" + Base,
-	"openssl":                   "quay.io/microshift/openssl" + Base,
+	"cli":             "quay.io/microshift/cli:" + Base,
+	"coredns":         "quay.io/microshift/coredns:" + Base,
+	"haproxy_router":  "quay.io/microshift/haproxy-router:" + Base,
+	"kube_rbac_proxy": "quay.io/microshift/kube-rbac-proxy:" + Base,
+	"odf_topolvm":     "quay.io/microshift/odf-topolvm-rhel8" + Base,
+	"openssl":         "quay.io/microshift/openssl" + Base,
+	// TODO (copejon) add snapshotter
```

Comment:
I think most of the whitespace changes in this file are triggered by this comment. Maybe Jira is a better place to track the todo?